.pre-commit-config.yaml

```diff
@@ -209,21 +209,24 @@ repos:
     - tests/
     pass_filenames: false
 
-- repo: local
+- repo: https://github.com/pre-commit/mirrors-pylint
```

Comment:
```suggestion
- repo: https://github.com/pre-commit/mirrors-pylint.git
```
(consistency)

Reply:
it looks like the entry is now:

```
- repo: https://github.com/pycqa/pylint.git
```

Reply:
It is :+1: 
(although, I'd prefer to also keep the case of `PyCQA` in the URL too but that's minor)